Assets/MixedRealityToolkit/Utilities/Editor/ScriptingUtilities.cs

```diff
@@ -1,48 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-#if UNITY_EDITOR
-using UnityEngine;
-using UnityEditor;
+using System.Collections.Generic;
 using System.IO;
+using UnityEditor;
+using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
 {
     /// <summary>
     /// A set of utilities to configure script compilation. 
     /// </summary>
-    public class ScriptingUtilities
+    public static class ScriptingUtilities
     {
         /// <summary>
         /// Appends a set of symbolic constant definitions to Unity's Scripting Define Symbols for the
         /// specified build target group.
         /// </summary>
-        /// </summary>
-        /// <param name="fileName">The name of an optional file locate before appending.</param>
-        /// <returns>
         /// <param name="targetGroup">The build target group for which the sybmols are to be defined.</param>
         /// <param name="symbols">Array of symbols to define.</param>
-        /// <remarks>
-        /// To always append the symbols, pass null (or the empty string) for the fileName parameter.
-        /// </remarks>
         public static void AppendScriptingDefinitions(
```

Comment:
>AppendScriptingDefinitions [](start = 27, length = 26)

Do we have an opinion on whether or not these types of functions constitute "public API" as well? i.e. otherwise this is also another type of breaking change.

Reply:
It has been documented as a breaking change in the release notes in this PR. I would be happy to put back the old signature / implementation and deprecate.

Reply:
This is, indeed, a breaking change. The API was introduced in 2.2.0 and, to the best of my knowledge, is used in one MRTK file that has since been updated.

I will investigate a mitigation and apply as appropriate.

Reply:
renamed the new class ScriptUtilities, restored and marked obsolete the previous implementation